traject.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "yell"                        # logging
   spec.add_dependency "dot-properties", ">= 0.1.1"  # reading java style .properties
   spec.add_dependency "httpclient", "~> 2.5"
+  spec.add_dependency 'marc-fastxmlwriter', '>=0.1' # fast marc->xml
```

Comment:
If we're releasing a traject 2.0 depending on it, do you think it's time to call marc-fastxmlwriter 1.0?  Then we could semver our dependency to `~> 1.0` (ie, 1.x). 

Otherwise, we should probably dependency on  `">= 0.1", "< 2"` for future-safeness, rather than any version at all greater than 0.1.